arkane/gaussian.py

```diff
@@ -262,13 +262,23 @@ def load_energy(self, zpe_scale_factor=1.):
         CBS-QB3 value.
         """
         e_elect, e0_composite, scaled_zpe = None, None, None
-
+        elect_energy_source = ''
         with open(self.path, 'r') as f:
             line = f.readline()
             while line != '':
 
                 if 'SCF Done:' in line:
                     e_elect = float(line.split()[4]) * constants.E_h * constants.Na
+                    elect_energy_source = 'SCF'
+                elif 'MP2 =' in line:
```

Comment:
I'm going to drop this commit because I am not sure I even have the files or knowledge anymore to unittest it. I've been using molpro for all my calculations, so it also doesn't effect my work at all. I think I might have written it to help someone else with some calculation, but this was too long ago and I can't really remember.

Reply:
before you drop it, maybe @dranasinghe can take a look?

Reply:
It'd be useful to fix this bug (if it is still a bug), I just completely forgot why I made the change and what exactly it fixes.

I dropped from this PR, though we can add it back or make a new PR. Here's access to the commit: https://github.com/ReactionMechanismGenerator/RMG-Py/commit/ab0705cf9ed0f1e44d05bdc7cb8510d6efdc8a70